bindings/python/IK/src/IntegrationBasedIK.cpp

```diff
@@ -44,6 +45,35 @@ void CreateIntegrationBasedIK(pybind11::module& module)
     py::class_<IntegrationBasedIK, //
                ILinearTaskSolver<IKLinearTask, IntegrationBasedIKState>>(module,
                                                                          "IntegrationBasedIK");
+
+    py::class_<IntegrationBasedIKProblem>(module, "IntegrationBasedIKProblem")
+        .def_readwrite("varable_handler", &IntegrationBasedIKProblem::variableHandler)
```

Comment:
Meta comment. I noticed we have some consistency issue with ``System::VariablesHandler``. At a certain point we should converge with the use of either ``VariableHandler`` or ``VariablesHandler``.

Reply:
Done in a3eda25f9cb3556cb558d3feb91ccee500c53f00 and 7bafd2d8720cd0438e453abada1b4e3cda98e97b